lighthouse-core/lib/sd-validation/sd-validation.js

```diff
@@ -64,12 +62,73 @@ module.exports = async function validate(textInput) {
   if (schemaOrgErrors.length) {
     return schemaOrgErrors.map(error => {
       return {
-        validator: /** @type {ValidatorType} */ ('schema-org'),
+        validator: /** @type {LH.StructuredData.ValidatorType} */ ('schema-org'),
         path: error.path,
         message: error.message,
+        lineNumber: error.path ? getLineNumberFromJsonLDPath(inputObject, error.path) : null,
+        invalidTypes: error.invalidTypes,
       };
     });
   }
 
   return [];
 };
+
+/**
+ * @param {*} obj
+ * @param {string} path
+ * @returns null | number - line number of the path value in the prettified JSON
+ */
+function getLineNumberFromJsonLDPath(obj, path) {
+  // To avoid having an extra dependency on a JSON parser we set a unique key in the
```

Comment:
has that shipped sailed with `jsonlint-mod` is there any way to leverage that?

Reply:
also deja vu, I feel like I've reviewed this before so sorry if you've answered this in the previous PR

Reply:
> has that shipped sailed with jsonlint-mod is there any way to leverage that?

I don't see a direct way to leverage jsonlint-mod here. It just uses Douglas Crockford's JSON parser.

I guess we could parse it ourselves or find a small parser that we can include?

> also deja vu, I feel like I've reviewed this before so sorry if you've answered this in the previous PR

Yup, I sent you a [pre-PR](https://github.com/mattzeunert/lighthouse/pull/1) to get some initial feedback.